mawabot/cogs/info.py

```diff
@@ -127,8 +137,26 @@ def _get_members(self, guild, names):
             name = f'{user.name}#{user.discriminator}'
             embed.set_author(name=name)
             embed.set_thumbnail(url=user.avatar_url)
-            embed.add_field(name='Status:', value=f'`{user.status}`')
+            if isinstance(user, discord.Member):
+                embed.add_field(name='Status:', value=f'`{user.status}`')
             embed.add_field(name='ID:', value=f'`{user.id}`')
+
+            # Get connected accounts
+            if profile is not None:
+                if profile.connected_accounts:
+                    accounts = []
+
+                    for account in profile.connected_accounts:
+                        if account['type'] == 'steam':
+                            url = f'https://steamcommunity.com/profiles/{account["id"]}'
+                            accounts.append(f'[{account["name"]}]({url})')
+                        elif account['type'] == 'twitch':
+                            url = f'https://www.twitch.tv/{account["name"]}'
+                            accounts.append(f'[{account["name"]}]({url})')
```

Comment:
As we add more types of supported connected accounts, we could change this into a dict.

```py
ACCOUNT_TYPES = {
    'steam': lamdba acct: f'https://steamcommunity.com/profiles/{acct["id"]}',
    'twitch': lamdba acct: f'https://www.twitch.tv/{acct["name"]}',
}
```


Just a thought I had.